sdk/identity/identity/test/internal/node/clientCertificateCredential.spec.ts

```diff
@@ -104,7 +104,7 @@ describe("ClientCertificateCredential (internal)", function() {
       // Emptying the token cache before we start.
       const tokenCache = new TokenCachePersistence(tokenCachePersistenceOptions);
       const persistence = await tokenCache.getPersistence();
-      persistence?.save("");
+      persistence?.save("{}");
```

Comment:
What was the reason for this change?

Reply:
This is the bug fix, writing an empty string causes an error, this error didn't bubble up on Node until Node 15.

Reply:
Oh, so the bug was in the test code?

Reply:
yup yup, only in the test code.